components/layout/AppMobileNavbar.vue

```diff
@@ -118,6 +123,9 @@ export default {
       this.$store.commit('alert/clear');
       context.toggleOpen();
     },
+    to(route) {
```

Comment:
task(blocking): if the div is an app-link component this function would be obsolete, also, i was unable to find the helper function for sameRouteReload. This would cause a js error if the link is clicked, please remove it.

Reply:
Fixed